lib/utils/env_vars.js

```diff
@@ -13,11 +13,16 @@ const env_vars = {
   PLAYGROUND_EMAIL: process.env.PLAYGROUND_EMAIL,
   PLAYGROUND_PASSWORD: process.env.PLAYGROUND_PASSWORD,
   BASE_URL: process.env.BASE_URL,
-  BRANCH_NAME: process.env.BRANCH_PATH && process.env.BRANCH_PATH.split('/')[1] || 'test',
   REPO_URL: process.env.REPO_URL,
   DEBUG: process.env.DEBUG,
 };
 
+// Path could be 'refs/branch_name' or 'refs/head/branch_name' and maybe other things
+// Assume branch name is last part of branch path
+let split = process.env.BRANCH_PATH.split('/');
```

Comment:
Maybe this should be in the if block following. In my Chrome console, the following lead to errors: 

```javascript
x = null 
x.split('/')
// ERROR 
x = undefined 
x.split('/')
// ERROR 
```

Reply:
Makes sense. I was thinking we'd want it to error if the branch path was messed up, but we're providing a fallback already. There are probably pros and cons about _that_ to talk about too, but I'm not sure how crucial that conversation is, so we might as well stick with what we're doing now.

Reply:
I suppose we then don't need the check on line 23 to check for the existence of the item on the list as even a single item will be valid.